### ISSUE TITLE
text_extraction_with_bert.py model doesn't learn

### ISSUE THREAD

**Issue by matthiaspetry** [keras-team/keras-io#1441]
Hey,

i tried to fine tune the bert model for text extraction with the provided notebook on google colab but the model doesn't learn

Epoch 1/3
WARNING:tensorflow:Gradients do not exist for variables ['tf_bert_model/bert/pooler/dense/kernel:0', 'tf_bert_model/bert/pooler/dense/bias:0'] when minimizing the loss. If you're using `model.compile()`, did you forget to provide a `loss` argument?
WARNING:tensorflow:Gradients do not exist for variables ['tf_bert_model/bert/pooler/dense/kernel:0', 'tf_bert_model/bert/pooler/dense/bias:0'] when minimizing the loss. If you're using `model.compile()`, did you forget to provide a `loss` argument?
WARNING:tensorflow:Gradients do not exist for variables ['tf_bert_model/bert/pooler/dense/kernel:0', 'tf_bert_model/bert/pooler/dense/bias:0'] when minimizing the loss. If you're using `model.compile()`, did you forget to provide a `loss` argument?
WARNING:tensorflow:Gradients do not exist for variables ['tf_bert_model/bert/pooler/dense/kernel:0', 'tf_bert_model/bert/pooler/dense/bias:0'] when minimizing the loss. If you're using `model.compile()`, did you forget to provide a `loss` argument?
323/323 [==============================] - 30s 58ms/step

epoch=1, exact match score=0.00
1346/1346 - 434s - loss: 11.8147 - activation_loss: 5.9112 - activation_1_loss: 5.9035 - 434s/epoch - 323ms/step
Epoch 2/3
323/323 [==============================] - 17s 42ms/step

epoch=2, exact match score=0.00
1346/1346 - 278s - loss: 11.9050 - activation_loss: 5.9524 - activation_1_loss: 5.9526 - 278s/epoch - 207ms/step
Epoch 3/3
323/323 [==============================] - 14s 41ms/step

epoch=3, exact match score=0.00
1346/1346 - 275s - loss: 11.9038 - activation_loss: 5.9518 - activation_1_loss: 5.9520 - 275s/epoch - 205ms/step
<keras.callbacks.History at 0x788ecebd5ed0>

**Comment by sampathweb** [comment]
@matthiaspetry - Can you provide a sample Colab notebook with a public dataset or the dataset used in the example?
